### ISSUE TITLE
support for translation-plugin / Startpage to first

### ISSUE THREAD

**Issue by frank-w** [splitbrain/dokuwiki-plugin-dw2pdf#318]
Hi,

please use language selected in translation-Plugin for Link-Text.

Startpage (default "start") should be the first page when exporting a full Namespace

regards Frank

**Comment by Klap-in** [comment]
Request: please one issue per issue, thanks!

About language selection:
For example current link is "Export namespace "test:" to file pdftitle.pdf"


Do you mean that at page "fr:ns1:page" you see the link:
 "Exporter la catégorie  « test: » dans le fichier pdftitle.pdf."

At page "en:ns1:page" you see the link:
 "Export namespace "test:" to file pdftitle.pdf."

etc.

Or do you mean that the namespace of the export should determine the title with languagecode used in the namespace that should be exported?

So at page "ns1:page" you see the link:
 "Exporter la catégorie  « fr:test: » dans le fichier pdftitle.pdf."

At page "ns1:page" you see the link:
 "Export namespace "en:test:" to file pdftitle.pdf."

If this is not corresponding with your idea, could you make an example which shows the idea?

**Comment by frank-w** [comment]
Yes, i mean the namespace-export-link should be in the language of that namespace. Currently the link-title is always the main-language of the wiki. The pdf itself uses the right language as i see

**Comment by Klap-in** [comment]
aha, so if you make somewhere a link to export "fr:test", then you expect a french url?
e.g. "Exporter la catégorie « fr:test: » dans le fichier pdftitle.pdf."

Do you also expect that this url is french if it is referred from a page in for example a Swedish namespace?

**Comment by frank-w** [comment]
Right, 

here my example (english namespace in german wiki). Here i changed lang/de/lang.php to get in both namespaces a description that is nearly understandable by both languages.

http://www.fw-web.de/dokuwiki/doku.php?id=en/bpi-r2/start

**Comment by frank-w** [comment]
The link should be in language of the page where it is located...the target is shown in link....

**Comment by frank-w** [comment]
Is clear what i mean? Have you take a look in my wiki?

**Comment by Klap-in** [comment]
Also related to #438
